### PR TITLE
Add cookiecutter as a dependency and split 6_Extending_the_toolkit in two parts

### PR DESCRIPTION
**Pull request by fbudin69500**

### CODE REVIEW

File: 6_Extending_the_Toolkit-Part2.ipynb [hunk 1]
```diff
@@ -8,203 +8,7 @@
     }
```
Comment (thewtex) [inline 6_Extending_the_Toolkit-Part2.ipynb:8]: <p>I think we can just mention Azure Pipelines now</p>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/2019-03-13-KRSCourseInBiomedicalImageAnalysisAndVisualization/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (thewtex) [inline 6_Extending_the_Toolkit-Part2.ipynb:8]: <p>This is great!</p><p><br></p><p>One more screenshot / discussion point would be helpful: the CDash page for the pull request... then C++ warnings, build errors, tests can be explained when you discuss it.</p>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/2019-03-13-KRSCourseInBiomedicalImageAnalysisAndVisualization/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (fbudin69500) [reply 6_Extending_the_Toolkit-Part2.ipynb:8]: <p>OK</p>

Comment (fbudin69500) [reply 6_Extending_the_Toolkit-Part2.ipynb:8]: <p>Makes sense</p>

**Pull request merged**